src/argilla/client/feedback.py

```diff
@@ -91,16 +90,7 @@ def responses_must_be_a_list(cls, v):
         return v
 
     class Config:
-        extra = Extra.forbid
-
-
-class OfflineFeedbackRecord(BaseModel):
-    id: Optional[str] = None
-    fields: Dict[str, str]
-    responses: List[ResponseSchema] = []
-    external_id: Optional[str] = None
-    inserted_at: Optional[str] = None
-    updated_at: Optional[str] = None
+        extra = Extra.ignore
```

Comment:
Can we show a warning if more properties than expected are passed?

Reply:
I don't think we can combine `Extra.ignore` with a `root_validator` to make sure that just the existing fields are provided, maybe @gabrielmbmb knows a cleaner way to do so?

Reply:
Also note that we want to ignore the extras, also the ones that come from Argilla for the moment i.e. `updated_at`, `created_at`, etc. fields that are not required/needed by the user, also to keep consistency for the records. So when we log records to Argilla and then we want to pull them, just the `FeedbackRecord` fields are kept from the API response